metricflow/test/integration/test_cases/itest_semi_additive_measure.yaml

```
@@ -53,4 +53,56 @@ integration_test:
     ON
       a.ds = b.ds
     GROUP BY
-      a.ds
\ No newline at end of file
+      a.ds
+---
+integration_test:
+  name: semi_additive_measure_query_with_join_linkable_specs
+  description: |
+    Tests semi-additive measure with a path hitting join linkable specs. 
+    There are cases (like this query where we have a non-local linkable spec "user__home_state_latest")
+    we are directly copying a MeasureSpec via constructing it. Previously, this led to a code path not passing all the attributes,
+    then somewhere along the path it would end up throwing a MeasureSpec not in [List of MeasureSpec] due to an attribute
+    being missing. This test case hits that path and ensures that we are copying the correct.  
```

Found some typos, but you may merge whenever you're ready!
```suggestion
    Tests semi-additive measure with a path hitting join linkable specs. 
    There are cases (like this query where we have a non-local linkable spec "user__home_state_latest") where
    we are directly copying a MeasureSpec via constructing it. Previously, this led to a code path not passing all the attributes,
    then somewhere along the path it would end up throwing a MeasureSpec not in [List of MeasureSpec] due to an attribute
    being missing. This test case hits that path and ensures that we are copying attributes correctly.  
```